tests/extendrtests/src/rust/Cargo.toml

```diff
@@ -16,7 +16,7 @@ crate-type = ["staticlib"]
 extendr-api = "*"
 
 [patch.crates-io]
-extendr-api = { git = "https://github.com/extendr/extendr" }
-extendr-engine = { git = "https://github.com/extendr/extendr" }
-extendr-macros = { git = "https://github.com/extendr/extendr" }
+extendr-api = { git = "https://github.com/clauswilke/extendr", branch = "reg-forwarding" }
+extendr-engine = { git = "https://github.com/clauswilke/extendr", branch = "reg-forwarding" }
+extendr-macros = { git = "https://github.com/clauswilke/extendr", branch = "reg-forwarding" }
```

Comment:
@clauswilke Should this have been merged? Looks like a temporary workaround.

Reply:
It's necessary, see #89. I'll fix with #91 as soon as GH Actions complete.

Reply:
👍 Ah okay, hadn't seen that issue yet.